### DISCUSSION TITLE
Zoom to specific lat/lon in COG viewer

### DISCUSSION THREAD

**Discussion by dchirst** [developmentseed/titiler#599] [Feature Request]
I've been using Titiler to serve some internal data to our developers, and some folks have requested the ability to zoom to a specific point from the COG viewer. 

This could be available as a couple of input boxes on the panel. 

Does this sound like an interesting enhancement? I'd be happy to write the code if it's of interest.

**Comment by vincentsarago** [comment]
Hi @dchirst any enhancements to the viewer are welcome 🙏 

Note: application viewers have been moved to https://github.com/developmentseed/titiler/tree/main/src/titiler/extensions/titiler/extensions

but now we also have simple `/map` viewer for all endpoints derived from TilerFactory https://github.com/developmentseed/titiler/blob/main/src/titiler/core/titiler/core/templates/index.html

**Reply by dchirst** [reply]
Sounds good, I'll have a look! Thank you
